opm/simulators/utils/readDeck.cpp

```diff
@@ -439,8 +439,8 @@ Opm::setupLogging(const int          mpi_rank_,
             : std::filesystem::current_path().generic_string();
     }
 
-    logFileStream << output_dir << "/" << baseName;
-    debugFileStream << output_dir << "/" << baseName;
+    logFileStream << fpath.parent_path().string() << "/" << fpath.stem().string();
+    debugFileStream << fpath.parent_path().string() << "/" << fpath.stem().string();
```

Comment:
I think this will write those streams to the deck location and not the output directory?

I can reproduce the problem, and will take a look at this.

Reply:
> I think this will write those streams to the deck location and not the output directory?

Yes, this could potentially break the `--output-dir` option.  Ideally we'd use [`IOConfig::getFullBasePath()`](https://github.com/OPM/opm-common/blob/e111bf0e6e63e92664d1d43eafc9774788e5547b/src/opm/input/eclipse/EclipseState/IOConfig/IOConfig.cpp#L273) here, but we typically configure the logging system prior to forming the `EclipseState`.